projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -58,12 +51,37 @@ const BarChart: FC< BarChartProps > = ( {
 		[ showTooltip ]
 	);
 
-	const handleMouseLeave = useCallback( () => {
-		hideTooltip();
-	}, [ hideTooltip ] );
-
+	const {
```

Comment:
It would a lot clearer to keep those props destructuring as function param IMHO.